src/named.rs

```diff
@@ -605,26 +597,6 @@ impl NamedChain {
     }
 
     /// Returns the chain's blockchain explorer and its API (Etherscan and Etherscan-like) URLs.
-    ///
-    /// Returns `(API_URL, BASE_URL)`.
-    ///
-    /// All URLs have no trailing `/`
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use alloy_chains::NamedChain;
-    ///
-    /// assert_eq!(
-    ///     NamedChain::Mainnet.etherscan_urls(),
-    ///     Some(("https://api.etherscan.io/api", "https://etherscan.io"))
-    /// );
-    /// assert_eq!(
-    ///     NamedChain::Avalanche.etherscan_urls(),
-    ///     Some(("https://api.snowtrace.io/api", "https://snowtrace.io"))
-    /// );
-    /// assert_eq!(NamedChain::AnvilHardhat.etherscan_urls(), None);
-    /// ```
```

Comment:
these docs we want to keep

Reply:
Sorry, it got automatically deleted when running `cargo run fmt --all`, fixed now.